packages/flutter_localizations/lib/src/l10n/cupertino_en.arb

```diff
@@ -0,0 +1,77 @@
+{
+  "datePickerHourSemanticsLabelOther": "$hour o'clock",
+  "@datePickerHourSemanticsLabel": {
+    "description": "Accessibility announcement for the selected hour on a time picker such as '5 o'clock' or '5点'",
+    "plural": "hour"
+  },
+
+  "datePickerMinuteSemanticsOne": "1 minute",
+  "datePickerMinuteSemanticsOther": "$minute minutes",
+  "@datePickerMinuteSemanticsLabel": {
+    "description": "Accessibility announcement for the selected minute on a time picker such as '15 minutes' or '15分'",
+    "plural": "minute"
+  },
+
+  "datePickerDateOrder": "mdy",
+  "@datePickerDateOrder": {
+    "description": "The standard order for the locale to arrange day, month and year in a date. Options are dmy, mdy, ymd and ydm.",
+  },
+
+  "datePickerDateTimeOrder": "date_time_dayPeriod",
+  "@datePickerDateTimeOrder": {
+    "description": "The standard order for the locale to date, time and am/pm in a datetime. Options are date_time_dayPeriod, date_dayPeriod_time, time_dayPeriod_date and dayPeriod_time_date where 'dayPeriod' is am/pm.",
+  },
+
+  "anteMeridiemAbbreviation": "AM",
+  "@anteMeridiemAbbreviation": {
+    "description": "The abbreviation for ante meridiem (before noon) shown in the time picker.",
+  },
+
+  "postMeridiemAbbreviation": "PM",
+  "@postMeridiemAbbreviation": {
+    "description": "The abbreviation for post meridiem (after noon) shown in the time picker.",
+  },
+
+  "alertDialogLabel": "Alert",
+  "@alertDialogLabel": {
+    "description": "The accessibility audio announcement made when an iOS style alert dialog is opened."
+  },
+
+  "timerPickerHourLabelOne": "hour",
+  "timerPickerHourLabelOther": "hours",
+  "@timerPickerHourLabel": {
+    "description": "The label adjacent to an hour integer number in a countdown timer.",
+    "plural": "hour"
+  },
+
+  "timerPickerMinuteLabelOther": "min",
```

Comment:
minute?
I think the one case is needed here

This one should probably be "$minute minutes"

Reply:
so this one is a bit weird... it's not actually part of the same string as $minutes. See https://flutter.dev/images/widget-catalog/cupertino-timer-picker.png. It's in a separate widget. Its pluralization depends on its sibling widget but it's not one string.

Reply:
Does it? According to localizations_validator.dart, only Other is needed.

Reply:
Thanks for the pointer the screenshot, I didn't understand how this was going to be used. You're right that only the "Other" plural translation is needed. It's going to be tricky to explain what's needed to the translators however it's possible to include a screenshot. That will help.

Reply:
ya, I saw the screenshot mechanism on the console. I'll likely add all of them to remind them of the iOS context.